apps/reference/docs/guides/realtime/limits.mdx

```diff
@@ -0,0 +1,66 @@
+---
+id: limits
+title: Realtime Limits
+description: Realtime Rate Limiting
+---
+
+Since deploying a global multi-tenant cluster for Supabase Realtime, we've implimented some rate limits to protect our infrastructure.
```

Comment:
```suggestion
Since deploying a global, multi-tenant cluster for Supabase Realtime, we've implemented some rate limits to protect our infrastructure.
```

Reply:
> "Since deploying a global, multi-tenant cluster for Supabase Realtime" 

these "time based" statements age a lot in documents. It implies that we recently made a change to our infra, but at what point in the future do we change this? Better to word things to be timeless from the start

Also - I don't know if this is relevant for a developer. The cluster is plenty scalable, and the limits are about fair usage. I think we can just say that there are limits for each plan.

Reply:
Going to sit with this for a bit and see how it feels there:

> Supabase Realtime is a global multi-tenant cluster. We've implemented some rate limits to help ensure high availability for all customers.